### PR TITLE
roll engine to d48ba4c0349109c3b5c62e77cfe2d26e794aabcb

### PR DESCRIPTION
**Pull request by xster**

### CODE REVIEW

**Review by cbracken — APPROVED** [review]

**Comment by cbracken** [comment]
Windows build appears not to have happened -- lgtm once that's fixed (or roll to a later SHA).

**Comment by xster** [comment]
Seems like the engine bot flaked. Finished building. Rerunning tests

**Pull request merged**
